.travis.yml

```diff
@@ -8,13 +8,18 @@ before_install:
 - git remote set-branches origin master && git fetch
 - git checkout master
 - git checkout -
+
+before_script:
+- npm run nsp-check # -- --warn-only
```

Comment:
ok, I see. No one will ever see warnings in the CI build so they are not useful. I suggest adding a comment above this command that says "If this command fails and you can't fix it, file an issue and add an exception to .nsprc"